joern-cli/src/main/scala/io/joern/joerncli/JoernScan.scala

```diff
@@ -59,7 +60,11 @@ object JoernScan extends App with BridgeBase {
 
     opt[Unit]("dump")
       .action((_, c) => c.copy(dump = true))
-      .text("Dump available queries to file")
+      .text("Dump available queries to a temporary file")
```

Comment:
```suggestion
      .text("Dump available queries to a file")
```
I'd keep it as before - the file isn't automatically removed, at least not by us...

Reply:
:+1: good point, thx